frontend/src/views/account-mgmt/ManageUsersView.vue

```diff
@@ -276,6 +276,44 @@ export default {
     isExpenseAuthority(user) {
       return user.facilities.some((facility) => facility.isExpenseAuthority) ? 'Yes' : 'No'
     },
+
+    /**
+     * Deactivates a user if they are not the last Expense Authority for any facility.
+     */
+    deactivateUser(user) {
+      const { isLastExpenseAuthority, facilityNames } = this.isUserLastExpenseAuthority(user.contactId);
+      if (isLastExpenseAuthority) {
+        this.setFailureAlert(`Cannot deactivate user. They are the last expense authority for facilities: ${facilityNames.join(', ')}.`);
```

Comment:
Can you run your formatter on this file? There seems to be semicolons.

Reply:
Fixed.